### ISSUE TITLE
Option to prevent line wrap of variable

### ISSUE THREAD

**Issue by spacemanspiff2007** [cknd/stackprinter#28]
Hi, thank you for creating stackprinter! It is really nice.

I have a question:
I'd like to prevent the the line wrap of the variables.

e.g.
```
30      p_mock.assert_called_once = <method 'NonCallableMock.assert_called_once' of <MagicMock i
30                                   d='88760712'> mock.py:808>
```
to
```
30      p_mock.assert_called_once = <method 'NonCallableMock.assert_called_once' of <MagicMock id='88760712'> mock.py:808>
```

would it be possible to add a parameter to ``stackprinter.format`` that prevents this?

**Comment by cknd** [comment]
Hi! 
That should be possible. The wrapping happens [down here](https://github.com/cknd/stackprinter/blob/master/stackprinter/prettyprinting.py#L243) in a pretty printer routine that is itself pretty convoluted but [does already expose the linewrap argument](https://github.com/cknd/stackprinter/blob/bbbb3803cfa313c1cbfc6df929850bcc5dea2e6a/stackprinter/prettyprinting.py#L21-L48). So it's just a question of wiring this argument all the way up [from the call site](https://github.com/cknd/stackprinter/blob/master/stackprinter/frame_formatting.py#L172) to a top level config in `format()`

I'm pretty short on time currently, so I can't promise this will land quickly, though

**Comment by spacemanspiff2007** [comment]
I tried to implement it and it seems to work.
Could you check the implementation and see if I missed something?

**Comment by cknd** [comment]
merged #32

**Comment by cknd** [comment]
(..and released in https://pypi.org/project/stackprinter/0.2.4 :tada:)
